src/main/java/net/wurstclient/hacks/MobSpawnEspHack.java

```diff
@@ -52,6 +53,7 @@
 		"Draw distance", DrawDistance.values(), DrawDistance.D9);
 	private final SliderSetting loadingSpeed =
 		new SliderSetting("Loading speed", 1, 1, 5, 1, v -> (int)v + "x");
+	private final CheckboxSetting xRay = new CheckboxSetting("X-Ray", "Reveal out-of-sight spawn areas.", false);

```

Comment:
Should be called "Depth test" to make it consistent with other settings (e.g. .path).
Not a problem though, I can fix this later.